<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addNewTeamDiag</class>
 <widget class="QDialog" name="addNewTeamDiag">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>296</width>
    <height>246</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="idLabel">
         <property name="text">
          <string>ID : </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="idLineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="teamNameLabel">
         <property name="text">
          <string>Team name : </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="teamNameLineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="usernameLabel">
         <property name="text">
          <string>Username : </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="usernameLineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="passwordLabel">
         <property name="text">
          <string>Password : </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="passwordLineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="scoreLabel">
         <property name="text">
          <string>Score : </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="scoreLineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QPushButton" name="addPersonButton">
         <property name="text">
          <string>Add Person</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
